pkgs/tools/backup/hpe-ltfs/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchurl, fuse, icu, pkgconfig, libxml2, libuuid }:
+
+stdenv.mkDerivation rec {
+  version = "3.4.2_Z7550-02501";
+  pname = "hpe-ltfs";
+
+  src = fetchurl {
+    url = "https://github.com/nix-community/hpe-ltfs/archive/${version}.tar.gz";
+    sha256 = "0jwqpjlv92ld6v4ay3qjsjk94xq6gmk0kwdwz12bkaqaygymrcj7";
+  };
+
+  sourceRoot = "hpe-ltfs-3.4.2_Z7550-02501/ltfs";
+
+  buildInputs = [ 
+    fuse icu pkgconfig libxml2 libuuid
+  ];
+
+  meta = with stdenv.lib; {
+    description = "HPE LTO Filesystem (LTFS)-mount LTO tapes file filesystems";
+    homepage = https://support.hpe.com/hpesc/public/km/product/1009214665/Product;
+    license = licenses.gpl2;
```

Comment:
The repo where you've put this said GPLv2.1

Reply:
Also, you said this was for commercial products as well. Wouldn't that be the lesser?
(maybe my mistake)

Reply:
It is 2.1, see COPYING.LIB.

I guess that's gpl2Plus?

Reply:
It said in COPING.LIB
```
 GNU LESSER GENERAL PUBLIC LICENSE
 Version 2.1, February 1999
```

So I believe that's `lgpl21` in nixos because that's `LGPL-2.1-only`.

Reply:
README also has
```
This file is part of the HPE Linear Tape File System package.  HPE LTFS is
free software; you can redistribute it and/or modify it under the terms
of the GNU Lesser General Public License as published by the Free Software
Foundation, version 2.1 of the License.
```

Reply:
Wow, can't believe I missed that.  Thanks.